src/display/annotation_layer.js

```diff
@@ -1429,7 +1430,13 @@ class PushButtonWidgetAnnotationElement extends LinkAnnotationElement {
       container.title = this.data.alternativeText;
     }
 
-    this._setDefaultPropertiesFromJS(container);
+    if (this.enableScripting && this.hasJSActions) {
+      this._setDefaultPropertiesFromJS(container.lastChild);
+
+      container.lastChild.addEventListener("updatefromsandbox", jsEvent => {
```

Comment:
To make it a bit clearer which element this actually references, how about something like the following (assuming that I'm understanding the code correctly)?
```suggestion
      const linkElement = container.lastChild;
      this._setDefaultPropertiesFromJS(linkElement);

      linkElement .addEventListener("updatefromsandbox", jsEvent => {
```